.github/workflows/pr-checks.yml

```
@@ -0,0 +1,70 @@
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)